### PR TITLE
disable rsvp button on click

### PR DESCRIPTION
**Pull request by Carlosedo**
Solves: https://bloopers.thinkful.com/thinkful/pysplash/issues/267/events/latest/

Long time without using React so 🙏, double check haha

### CODE REVIEW

File: src/OpenSessionOverview/OpenSessionOverview.jsx [hunk 1]
```diff
@@ -98,6 +99,11 @@ class OpenSessionOverview extends React.Component {
     });
   }
 
+  _handleRSVPButtonClick(e, id) {
+    $(e.target).addClass('button__disabled');
+    handleRSVPClick(id);
```
Comment (learykara) [inline src/OpenSessionOverview/OpenSessionOverview.jsx]: I'd actually do this via the state.  Something like `this.setState({ requestPending: true })`, and use that on the component to add & remove the disabled class

**Comment by Carlosedo** [comment]
updated it @learykara 
please take another look 😄

**Comment by learykara** [comment]
what happens when the request is finished? does the component/page reload or does it stay in a loading state forever?

**Comment by Carlosedo** [comment]
When the request is finished `attending` is set to true and the button is hidden
https://github.com/Thinkful/ui/blob/master/src/OpenSessionOverview/OpenSessionOverview.jsx#L290

**Pull request merged**